spec/opportunities/org_admin_deletes_opportunity_spec.rb

```diff
@@ -0,0 +1,49 @@
+feature "org_admin deletes opportunity" do
+
+  let!(:user) { FactoryGirl.create(:user, volunteer: false) }
+  let!(:organization) { FactoryGirl.create(:organization) }
+  let!(:org_admin) { FactoryGirl.create(
+    :org_admin,
+    user: user,
+    organization: organization) }
+  let!(:opportunity) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.